client/src/app/services/api/api.service.ts

```diff
@@ -16,6 +16,10 @@ export class ApiService {
     return this.http.get(this.baseUrl + url, this.httpOptions);
   }
 
+  getStockHistoricalData(url: string): Observable<any> {
```

Comment:
This should be moved to Money-Tree/client/src/app/services/stock/stock.service.ts, we keep only general get, post, update, patch, delete request definitions for our api here.